src/styles.ts

```diff
@@ -35,12 +35,14 @@ export const styles = css`
 			var(--primary-color, #3f51b5)
 		);
 
-		display: block;
+		display: var(--cosmoz-input-display, block);
```

Comment:
Why do you need a var for these?

These styles are for the :host element, which is already stylable from outside of the component:

```css
cosmoz-input {
  display: inline-block;
  border: 1px solid red;
  border-radius: 10px;
}
```